Microsoft.Health.Fhir.Ingest.sln

```diff
@@ -116,6 +116,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.Inges
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.Ingest.Legacy", "src\lib\Microsoft.Health.Fhir.Ingest.Legacy\Microsoft.Health.Fhir.Ingest.Legacy.csproj", "{F35F00C4-4AB0-4E80-99F1-39F24948B7A8}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.Ingest.Console.Normalization", "src\console\Microsoft.Health.Fhir.Ingest.Console.Normalization\Microsoft.Health.Fhir.Ingest.Console.Normalization.csproj", "{C972DE01-7C95-414E-BED7-A2C322B78D32}"

```

Comment:
we can move the core code into 1 so that the downstream apps only need to consume 1 nuget package. went ahead and did that. there will still need to be 3 csproj files though. 1 for the core code, 1 for the normalization console app (imports the core code and runs a console app), and 1 for the fhirtranformation console app (imports the core code and runs a console app).